### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by keymone**
- address GH security alerts
- remove sqlalchemy dependency and related code
- import tron.yaml instead of yaml everywhere

### CODE REVIEW

**Comment by qui** [comment]
failing test :/ did the API of sqlalchemy change? we could also just delete that state store, unless you think it will be useful for local testing once we get rid of berkeleydb.

**Comment by keymone** [comment]
i'm in favor of removing sqlalchemy store

File: requirements.txt [hunk 1]
```diff
@@ -69,9 +69,6 @@ rsa==4.0
 s3transfer==0.2.0
 setuptools==41.0.1
 six==1.12.0
-SQLAlchemy==1.3.3
-task-processing==0.1.5
```
Comment (qui) [inline requirements.txt]: Don't we still need this?

Comment (keymone) [reply requirements.txt]: thanks!

**Pull request merged**